examples/launch_height_baselines.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+import logging
+from pathlib import Path
+
+from syne_tune.backend.local_backend import LocalBackend
+from syne_tune.optimizer.baselines import RandomSearch, BayesianOptimization, ASHA, MOBSTER
+from syne_tune.optimizer.baselines import PASHA, BORE  # noqa: F401
+from syne_tune.optimizer.schedulers.synchronous.hyperband_impl import SynchronousGeometricHyperbandScheduler  # noqa: F401
+from syne_tune.tuner import Tuner
+from syne_tune.search_space import randint
+from syne_tune.stopping_criterion import StoppingCriterion
+
+
+if __name__ == '__main__':
+    logging.getLogger().setLevel(logging.INFO)
+
+    random_seed = 31415927
+    max_steps = 100
+    n_workers = 4
+
+    config_space = {
+        "steps": max_steps,
+        "width": randint(0, 20),
+        "height": randint(-100, 100)
+    }
+    entry_point = Path(__file__).parent / "training_scripts" / "height_example" / "train_height.py"
+    mode = "min"
+    metric = "mean_loss"
+
+
+    schedulers = [
```

Comment:
In order to show people how they can set extra args, and also because we found that to work well, you could add

search_options = {'num_init_random': n_workers + 2}

and then pass that to BayesianOptimization and to MOBSTER. This is an example for using **kwargs there.

I think it is important people also understand how to configure the methods, if they want to.